### PR TITLE
feat: add multicall to widget

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

File: src/lib/components/Widget.tsx [hunk 1]
```diff
@@ -64,6 +67,15 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
   }
 `
 
+function Updater() {
```
Comment (ianlapham) [inline src/lib/components/Widget.tsx:70]: Nit - can we changer to `Updaters`

**Review by ianlapham — COMMENTED** [review]
Just small nit rest looks good

File: src/lib/components/Widget.tsx (hunk 1 above)
Comment (zzmp) [reply src/lib/components/Widget.tsx:70]: lol my personal nit was to change from `Updaters` to singular. I'll use the plural 😆

File: src/hooks/useContract.ts [hunk 2]
```diff
@@ -38,7 +38,8 @@ import useActiveWeb3React from 'hooks/useActiveWeb3React'
 import { useMemo } from 'react'
 import { NonfungiblePositionManager, Quoter, UniswapInterfaceMulticall } from 'types/v3'
 import { V3Migrator } from 'types/v3/V3Migrator'
-import { getContract } from 'utils'
+
+import { getContract } from '../utils'
```
Comment (zzmp) [inline src/hooks/useContract.ts:42]: This is changed as an alternative to adding `utils` to paths for `tsconfig.lib.ts`.

**Pull request merged**